### PR TITLE
SubTeam Auth: User Registration

### PR DESCRIPTION
**Pull request by DavidHODs**
This resolves issue #163

Users can now register after supplying required details

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (farouqu) [inline .vscode/settings.json:1]: Add your .vscode folder to .gitignore. It's not needed in the repo

**Comment by DavidHODs** [comment]
Alright

File: auth/register.go [hunk 2]
```diff
@@ -1 +1,63 @@
...
+	"fmt"
+	"net/http"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var client *mongo.Client
```
Comment (farouqu) [inline auth/register.go:15]: A client is already created which includes methods to perform basic queries in the utils folder. Check utils/db.go. You can easily import and use the functions

**Comment by DavidHODs** [comment]
Working on it

**Review by farouqu — CHANGES_REQUESTED** [review]
Check out [utils folder](github.com/zurichat/zc_core/utils/db.go) and use the database client there. It's uniform across the app.

**Comment by DavidHODs** [comment]
getMongoDbConnection() in utils is not designed to be visible outside that
package.

Was it purposely designed this way or I should change it and make it
applicable outside the package

**Comment by DavidHODs** [comment]
Please ignore my last mail

I've seen how the architecture connects with each other

**Pull request closed without merging**